packages/react-native-bridge/ios/GutenbergWebFallback/GutenbergWebSingleBlockViewController.swift

```diff
@@ -152,6 +158,7 @@ extension GutenbergWebSingleBlockViewController: WKNavigationDelegate {
         evaluateJavascript(jsInjection.injectWPBarsCssScript)
         evaluateJavascript(jsInjection.injectLocalStorageScript)
         onPageLoadScripts().forEach(evaluateJavascript)
+        onGutenbergReadyStyles().forEach(evaluateJavascript)
```

Comment:
👋 @SiobhyB - I notice that we're calling `onGutenbergReadyStyles` on both `didCommit` and `didFinish` for the `WKNavigationDelegate` - do you think perhaps we can get away with just using `didCommit`, or did you find `didFinish` was also necessary?

From my understanding, scripts are loaded in this sequence:

1. `didCommit` for the navigation delegate (this line)
2. `didFinish` for the navigation delegate
3. `onGutenbergReady()`

**This also brings up another question I wanted to ask:**

Should we rename `onGutenbergReadyStyles()` to something that matches the timing of when the script is loaded? I'm thinking this CSS injection works best when the page is loaded, but before it's ready. Some ideas I had were `onPageLoadStyles()` or `onGutenbergLoadStyles()`, but I'd love to hear your thoughts. Thanks!

Reply:
👋 @twstokes, thank you for your help here! 🙇‍♀️ 

> I notice that we're calling `onGutenbergReadyStyles` on both `didCommit` and `didFinish` for the `WKNavigationDelegate` - do you think perhaps we can get away with just using `didCommit`, or did you find `didFinish` was also necessary?

I had a feeling both were necessary as I saw `injectEditorCssScript` (which loads the CSS from [this file](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-bridge/common/gutenberg-web-single-block/editor-style-overrides.css)) was also called from both, with [the following comment](https://github.com/WordPress/gutenberg/blob/2b18866215a19f87f40d59b275e43b6b7b2b9781/packages/react-native-bridge/ios/GutenbergWebFallback/GutenbergWebSingleBlockViewController.swift#L165-L166) alongside `didFinish`: 

```
// Sometimes the editor takes longer loading and its CSS can override what
// Injectic Editor specific CSS when everything is loaded to avoid overwritting parameters if gutenberg CSS load later.
```

My thinking is that the CSS from `onGutenbergReadyStyles` serves basically the same purposes as the CSS from `injectEditorCssScript` (to hide or customize parts of the editor), except that its housed under the Mobile Gutenberg repository. So, I tried to follow the same patterns as `injectEditorCssScript`.

Does that make sense to you? Or do you think there's a case for removing `onGutenbergReadyStyles` from `didFinish`?

> Should we rename onGutenbergReadyStyles() to something that matches the timing of when the script is loaded? I'm thinking this CSS injection works best when the page is loaded, but before it's ready. Some ideas I had were onPageLoadStyles() or onGutenbergLoadStyles(), but I'd love to hear your thoughts. Thanks!

I think renaming makes sense! I like `onGutenbergLoadStyles`, as I feel like it'd work for both the `didCommit` and `didFinish` scenarios (if we do continue to call `onGutenbergLoadStyles` from both). I've gone ahead to update that in https://github.com/WordPress/gutenberg/pull/37221/commits/e8371aaadad36353c7cce82ef6d93056611b2d5b.

Reply:
> Does that make sense to you?

Sure does, and it won't hurt a thing to keep it. Sounds good to me to keep the patterns consistent @SiobhyB!